user_guide_src/source/testing/overview.rst

```diff
@@ -276,3 +276,23 @@ See :ref:`Testing Traits <testing-overview-traits>`.
 
 If you override the ``setUp()`` or ``tearDown()`` methods in your test, then you must call the ``parent::setUp()`` and
 ``parent::tearDown()`` methods respectively to configure the ``StreamFilterTrait``.
+
+.. _testing-cli-input:
+
+Testing CLI Input
+=================
+
+**PhpStreamWrapper** provides a way to write tests for methods that require user input,
+such as ``CLI::prompt()``, ``CLI::wait()``, and ``CLI::input()``.
```

Comment:
This can cause unexpected results if the user is not familiar with stream wrappers. Can you additional info that this is a stream wrapper for the php protocol and if you want to know more of it you can visit this link and that?

Reply:
Added explanations.

Reply:
Thanks!